core/files/findbugs-exclude.xml

```diff
@@ -2,4 +2,9 @@
 <FindBugsFilter>
     <!-- documentation about findbug's rule match clauses can be found here:
          http://findbugs.sourceforge.net/manual/filter.html -->
+         
+  <Match>
+    <Class name="com.graphhopper.resources.IsochroneResource" />
+    <Bug pattern="RV_EXCEPTION_NOT_THROWN" />
+  </Match>
```

Comment:
Somehow this is necessary as otherwise it fails and says:

```
[INFO] Total bugs: 1
[INFO] new IllegalArgumentException(String) not thrown in com.graphhopper.resources.IsochroneResource.doGet(HttpServletRequest, UriInfo, String, String, int, boolean, GHPoint, String, long, double) [com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource, com.graphhopper.resources.IsochroneResource] At IsochroneResource.java:[line 62]Another occurrence at IsochroneResource.java:[line 65]Another occurrence at IsochroneResource.java:[line 70]Another occurrence at IsochroneResource.java:[line 77]Another occurrence at IsochroneResource.java:[line 92]Another occurrence at IsochroneResource.java:[line 94]Another occurrence at IsochroneResource.java:[line 101]Another occurrence at IsochroneResource.java:[line 103]Another occurrence at IsochroneResource.java:[line 110]Another occurrence at IsochroneResource.java:[line 116] RV_EXCEPTION_NOT_THROWN
```